test/testenv/verificationutils.go

```diff
@@ -59,6 +59,25 @@ type PodDetailsStruct struct {
 	} `json:"status"`
 }
 
+// VerifyMCReady verify Standlone is in ReadyStatus and does not flip-flop
+func VerifyMCReady(deployment *Deployment, deploymentName string, monitoringConsole *enterprisev1.MonitoringConsole, testenvInstance *TestEnv) {
```

Comment:
Maybe Change the name to func MonitoringConsoleReady() to maintain consistency with other API